lab-tara/app/src/main/java/com/example/conwaysgameoflife/MainActivity.java

```diff
@@ -0,0 +1,44 @@
+package com.example.conwaysgameoflife;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+
+public class MainActivity extends AppCompatActivity {
+
+    private GameOfLife gameOfLife;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        ButterKnife.bind(this);
+
+        gameOfLife = (GameOfLife) findViewById(R.id.gameOfLife);
```

Comment:
This cast is not needed (findViewById is a generic, so it will return the right type)